py-polars/tests/conftest.py

```diff
@@ -42,6 +46,29 @@ def fruits_cars() -> pl.DataFrame:
     )
 
 
-@pytest.fixture
-def series() -> pl.Series:
-    return pl.Series("a", [1, 2])
+def _getattr_multi(obj: object, op: str) -> Any:
+    """ "
+    Allows `op` to be multiple layers deep, i.e. op="str.lengths" will mean we first
+    get the attribute "str", and then the attribute "lengths"
+    """
+    op_list = op.split(".")
+    return reduce(lambda o, m: getattr(o, m), op_list, obj)
+
+
+def verify_series_and_expr_api(
```

Comment:
Maybe add `**kwargs` as well?

Reply:
Done. Not using it at the moment, but may be useful indeed.